src/CookieStore.ts

```diff
@@ -12,6 +12,7 @@ interface ResponseLike {
 }
 type Store = Map<string, StoreEntry>
 type StoreEntry = Map<string, Cookie>
+type StringifiedCookie = Omit<Cookie, 'expires'> & { expires?: string };
```

Comment:
A union type without the `Omit` should produce the same result.

```ts
type StringifiedCookie = Cookie & { expires?: string }
```

Does it not?

Reply:
That would result in `expires` being defined as `undefined | string | Date`. It would not cause an error but I think it's slightly incorrect since expires can't be a Date when the value comes from the local storage.

Reply:
You're right, thanks for the explanation :)